java/BUILD_MANUAL.md

```diff
@@ -61,7 +61,11 @@ The total amount of all invoices in USD is: 1016.04
 ### Gradle
 #### Build Command
 ```shell
-gradle clean build 
+# Option 1: Gradle is already installed on your machine 
+> gradle clean build 
+
+# Option 2: Gradle is NOT installed on your machine 
+> gradlew clean build
```

Comment:
I'm not sure we can assume that the user's current working directory is in the path. For this reason a common recommendation when running gradlew is to type ./gradlew (...) which works in all cases, assuming of course that the user cd-ed previously in the java directory.

Reply:
If I understood well, you prefer to replace the "gradlew clean build" with "./gradle clean build". 
I am ok with that, however, we get another problem because on windows we need to use "./" and on Linux & Mac we need to use ".\"
Do we need to include both? or do you think it will be sufficient to mention just one?

Reply:
What I mean is to replace "gradlew clean build" with "./gradlew clean build".
The "gradle clean build" should not be changed as the assumption here is that if gradle is already installed on the user's machine, it's in the path already.
About windows users, they should run .\gradlew.bat so anyway the command will be different from unix family OS's

Reply:
I modified the text as suggested